plugin/lspconfig.lua

```diff
@@ -28,23 +28,48 @@ local lsp_complete_configured_servers = function(arg)
   end, util.available_servers()))
 end
 
-local lsp_get_active_client_ids = function(arg)
+local lsp_get_active_clients = function(arg)
   local clients = vim.tbl_map(function(client)
-    return ('%d:%s'):format(client.id, client.name)
+    return ('%s'):format(client.name)
```

Comment:
why remove id ? if there are multiple servers which does not support workspace folders, it will be difficult to distinguish which one want to stop

Reply:
Are saying that its possible to receive multiple entries like:
- 1 lua_ls
- 2 lua_ls 

If that is the case than I will revert this.

Is there a way to show the completion with the id and name but on complete only pass the id so the user
only needs to hit enter?

Reply:
Are saying that its possible to receive multiple entries like:
- 1 lua_ls
- 2 lua_ls 

If that is the case than I will revert this.

Is there a way to show the completion with the id and name but on complete only pass the id so the user
only needs to hit enter?

Reply:
revert .  if you want complete with input id need imporve  current custom complete  and split cmdline content to get id then complete with server name

Reply:
there still not revert.

Reply:
I believe it's best this way. This enhances the normal case. If a user needs to close a specific id he needs to know it using LspInfo anyway.

Reply:
nope